langchain/text_splitter.py

```diff
@@ -1056,3 +1057,97 @@ def __init__(self, **kwargs: Any) -> None:
         """Initialize a LatexTextSplitter."""
         separators = self.get_separators_for_language(Language.LATEX)
         super().__init__(separators=separators, **kwargs)
+
+
+class GrobidSplitter:
```

Comment:
this doesnt use grobid at all?

Reply:
it doesn't appear to use Grobid, but accepts the XML produced by Grobid.